docs/node/migrations/offline-migrations.mdx

```diff
@@ -0,0 +1,142 @@
+It is recommended that only one validator operator is responsible for generating, updating, and distributing the genesis state file to all other validators. This ensures that all validators are working from the same, consistent configuration.
+:::
+
+Distribute the `kwildb-snapshot.sql.gz` and the `genesis.json` files required to start the new network. 
+
+## Start New Network
+
+### Install Kwild Binaries
+
+Install the new version of the `kwild` binaries. Refer to the [Installation](/docs/daemon/installation) documentation for more details on how to install the `kwild` binaries.
+
+### Setup Root directory
+
+Copy the `kwildb-snapshot.sql.gz`, `genesis.json` files downloaded [above](/docs/node/migrations/offline-migrations#create-network-snapshot) to the new node's root directory. 
+
+:::note
+Ensure that the `private_key` corresponding to the validator node are securely copied to the new node's root directory.
+:::
+
+```
+root_dir
+├── config.toml
+├── private_key
+├── genesis.json
+├── snapshot.sql.gz
+├── abci/
+```
+
+### Configure Nodes To Use Snapshots
+
+Configure `config.toml` with required settings, refer to the [Node Configuration](/docs/daemon/config/settings) documentation for more details.
+
+To initialize the node with the snapshot data, set the `genesis_state` path in `config.toml` to the location of the downloaded snapshot file. 
+
+```TOML
+#######################################################################
+###                      App Config Options                         ###
+#######################################################################
+
+[app]
+
+# Path to the snapshot file to restore the database from.
+# Used during the network migration process.
+genesis_state = "/path/to/snapshot/file"
+```
+
+:::info
+To start a new network with the snapshot data, the genesis file must have the `app_hash` field set; otherwise, the network will start with an empty state.
+The node will fail to start if the `app_hash` field is set in the `genesis.json` file but the `genesis_state` path is not provided in the `config.toml` file. 
+:::
+
+### Start Kwild Nodes
+
+Once the genesis and config files are configured to use snapshots, start the node using the following command:
+
+```bash
+# use -h for more options. 
+kwild -r /path/to/node/root/dir
+```
+
+:::tip
+If your `genesis.json` file has multiple validators, you will need to start >2/3 of the validators before the network can start producing blocks.
+:::
+
+## Nodes Catching Up
+
+Follow the same steps described in the [Start New Network](#start-new-network) section to configure and join the network. Nodes can either sync with the network using `statesync` or `blocksync`.
+
+If the node uses `blocksync`, the `genesis_state` path is mandatory to ensure that the node initializes with the same genesis initial state and replay all the historical blocks. If the node doesn't start from the same initial state, it will eventually fork itself off the chain during blocksync. 
```

Comment:
> If the node uses `blocksync`, the `genesis_state` path is mandatory to ensure that the node initializes with the same genesis initial state and replay all the historical blocks. If the node doesn't start from the same initial state, it will eventually fork itself off the chain during blocksync. 

This feels potentially a bit too in-the-weeds for most users. Instead, I think you could just say something like:

**All nodes using `blocksync` need to have the genesis state locally** to sync with the chain. Nodes using `statesync` do not need to download the genesis state locally.

Reply:
We should also link to docs for both of these